### PR TITLE
Code for username proper format validation

### PR DESCRIPTION
**Pull request by chairana**
Code for validating the username provided by user.

### CODE REVIEW

File: src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java [hunk 1]
```diff
@@ -8,7 +11,10 @@ public boolean ValidateUserCredentials(String username, String password){
 
     private boolean ValidateUserName(String username){
         // Todo: - Check the username is in the proper format.
-        return false;
```
Comment (IronMan0325) [inline src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java:11]: Remove the return false statement.

File: src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java [hunk 2]
```diff
@@ -8,7 +11,10 @@ public boolean ValidateUserCredentials(String username, String password){
 
     private boolean ValidateUserName(String username){
         // Todo: - Check the username is in the proper format.
```
Comment (IronMan0325) [inline src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java]: Is todo still required after writing the code?

**Review by IronMan0325 — CHANGES_REQUESTED** [review]

File: src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java (hunk 1 above)
Comment (chairana) [reply src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java:11]: I will remove this return statement.

File: src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java (hunk 2 above)
Comment (chairana) [reply src/main/java/com/Setu/SetuHomePageService/Authentication/UserLogIn.java]: I will remove those comments.

**Review by IronMan0325 — APPROVED** [review]

**Pull request merged**